Zotero/Models/Defaults.swift

```diff
@@ -69,6 +69,12 @@ final class Defaults {
     @UserDefault(key: "QuickCopyAsHtml", defaultValue: false, defaults: .standard)
     var quickCopyAsHtml: Bool
 
+    @UserDefault(key: "TrashAutoEmptyDayThreshold", defaultValue: 30, defaults: .standard)
+    var trashAutoEmptyThreshold: Int
+
+    @UserDefault(key: "TrashLastAutoEmptyDate", defaultValue: .distantPast, defaults: .standard)
+    var lastAutoEmptyDate: Date
```

Comment:
Why not keep the pattern?

```suggestion
    var trashLastAutoEmptyDate: Date
```

Reply:
Changed fab280cb166d1f3787bd44eeab8f136df4332457